src/proc_image_processing/cpu/filters/detectors/center_coffin_detector.h

```diff
@@ -40,56 +40,60 @@ namespace proc_image_processing {
                 if (image.channels() != 1) cv::cvtColor(image, image, CV_BGR2GRAY);
 
                 PerformanceEvaluator timer;
-                timer.UpdateStartTime();
+                timer.resetStartTime();
 
                 contourList_t contours;
 
-                RetrieveAllContours(image, contours);
+                retrieveAllContours(image, contours);
                 ObjectFullData::FullObjectPtrVec objVec;
-                for (int i = 0, size = contours.size(); i < size; i++) {
```

Comment:
I'll bet vector.size() is as efficient as reading a local var. In that case it's ok to change it.

Reply:
Actually it's not why I changed this, it's caus of long to int conversion...
![image](https://user-images.githubusercontent.com/32100830/126249050-ee14f939-0a0c-46d6-9b21-f88b4e8d4ca9.png)

Reply:
![image](https://user-images.githubusercontent.com/32100830/126249079-bf369c23-d136-4381-8a34-d75af4c8c690.png)